compiler/rustc_codegen_llvm/src/llvm_util.rs

```diff
@@ -398,7 +398,10 @@ fn print_target_features(out: &mut dyn PrintBackendInfo, sess: &Session, tm: &ll
         "crt-static",
         "Enables C Run-time Libraries to be statically linked",
     )]);
+    rustc_target_features.sort();
+
     llvm_target_features.retain(|(f, _d)| !known_llvm_target_features.contains(f));
+    llvm_target_features.sort();
```

Comment:
could the llvm target features already be sorted? there's a binary search just above

Reply:
Yes, `llvm_target_features` appears to be sorted.

Turns out there is no regression test for the output. Probably because it is highly target specific and probably would be a pain to maintain.

Reply:
We do have a few that check `--print` and `-Z` output so it was still worth a shot, but they should have failed locally.